package.json

```diff
@@ -28,11 +28,7 @@
     },
     "patchedDependencies": {
       "wasi-js@1.7.3": "patches/wasi-js@1.7.3.patch",
-      "@aws-sdk/is-array-buffer@3.201.0": "patches/@aws-sdk__is-array-buffer@3.201.0.patch",
-      "@aws-sdk/util-buffer-from@3.208.0": "patches/@aws-sdk__util-buffer-from@3.208.0.patch",
       "jsii@5.0.11": "patches/jsii@5.0.11.patch",
-      "@aws-sdk/util-utf8-node@3.259.0": "patches/@aws-sdk__util-utf8-node@3.259.0.patch",
-      "@aws-sdk/util-utf8-browser@3.259.0": "patches/@aws-sdk__util-utf8-browser@3.259.0.patch",
```

Comment:
Nice, super glad to be able to remove this patch.
Could you delete the associated files in `patches` as well?

Reply:
Done [f4d1ff2](https://github.com/winglang/wing/pull/4720/commits/f4d1ff24921eb87b2b91f3b307697154f7813b0c)